### DISCUSSION TITLE
How to use vuestic-ui in nuxt 3

### DISCUSSION THREAD

**Discussion by havormark** [nuxt/nuxt#17572] [Questions] [answered]
Hi can anyone please help me to setup vuestic-ui on nuxt 3

**Comment by wobsoriano** [comment]
I am close to making it work.. just need to fix the error `HTMLInputElement is not defined`

```ts
// nuxt.config.ts
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
    css: ['vuestic-ui/dist/vuestic-ui.css'],
    meta: {
        link: [
            { src: 'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;1,700&display=swap', rel: 'stylesheet' },
            { src: 'https://fonts.googleapis.com/icon?family=Material+Icons', rel: 'stylesheet' },
        ]
    },
    build: {
        transpile: ["tslib", "@popperjs"],
    },
})
```

```ts
// plugins/vuestic.ts
import { VuesticPlugin } from 'vuestic-ui'
  
// Add tree-shaking here if you want

export default defineNuxtPlugin(nuxtApp => {
  nuxtApp.vueApp.use(VuesticPlugin)
})
```

**Reply by havormark** [reply]
Thanks. I'll be waiting for your fix then

**Reply by wobsoriano** [reply]
So the problem is probably with popper.. would be nice if you can help me fix

**Reply by havormark** [reply]
I tried but couldn't find any fix for it

**Comment by havormark** [comment]
It would be great to have it work with nuxt 3 though

**Comment by asvae** [comment]
Added [an issue](https://github.com/epicmaxco/vuestic-ui/issues/1487) on vuestic side.
We're currently focusing on nuxt support actually :).

**Comment by Derranion** [comment, marked as answer]
Hey guys @havormark @wobsoriano 

Vuestic UI is now officially supports Nuxt :)

Here is how to install it:
https://vuestic.dev/en/getting-started/nuxt

**Reply by wobsoriano** [reply]
Awesome
